api/models.py

```diff
@@ -71,10 +71,21 @@ def region_name(self):
 def logo_document_path(instance, filename):
     return 'logos/%s/%s' % (instance.iso, filename)
 
+class CountryType(IntEnum):
+    '''
+        We use the Country model for some things that are not "Countries".
+        For eg. IFRC (Not a country) or groups of countries like Southeast Asia (A region, not a country)
+    '''
+    NOT_COUNTRY = 0
+    COUNTRY = 1
+    GROUPING = 2
```

Comment:
Is `grouping` the best way? Or should we store actual admin level type?

Reply:
Yea, was not sure what to call this case -- so, we have entries that are Countries - eg. India. This is straightforward as type `1` above.

Groupings are things like:
 - `Andean Country Cluster`
 - `Asia Pacific Region`
 - `Central Asia Country Cluster`
 - `English Speaking Caribbean Country Cluster`

I'm not sure these fit neatly into admin level definitions either. Perhaps we can go with `Cluster` to be closer to how a lot of these are called in the backend?

@ElsaRaunioIFRC - do you have an idea for what "type" we should assign to things like the examples above?